### PR TITLE
Interfaces

### PR DESCRIPTION
**Pull request by selmaT273**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +18,17 @@ The contents of this lab will plan and build out what animals live in the zoo. I
...
+
+## Interface
+- An interface is a way to group types together by sharing the same kind of inheritance among them. It defines a role that an object can play by specifying a thing that an object is able to do. 
+
+## Interface Implementation
+- In my project, there are currently 2 interfaces: IEatFresh and ISpeakAlot. 
+### IEatFresh
+- This interface allows specific concrete animals to eat fresh food if they inherit from the IEatFresh interface.
+### ISpeakAlot
+- This interface allows some of the concrete animals to SpeakAlot if they inherit from the ISpeakAlot class.
```
Comment (dahlbyk) [inline README.md]: Language nitpick:
```suggestion
- This interface allows some of the concrete animals to SpeakAlot if they implement the ISpeakAlot interface.
```

File: README.md [hunk 2]
```diff
@@ -18,3 +18,17 @@ The contents of this lab will plan and build out what animals live in the zoo. I
...
+
+# Lab 07: Interfaces
+
+## Interface
+- An interface is a way to group types together by sharing the same kind of inheritance among them. It defines a role that an object can play by specifying a thing that an object is able to do. 
+
+## Interface Implementation
+- In my project, there are currently 2 interfaces: IEatFresh and ISpeakAlot. 
+### IEatFresh
+- This interface allows specific concrete animals to eat fresh food if they inherit from the IEatFresh interface.
```
Comment (dahlbyk) [inline README.md]: Language nitpick:
```suggestion
- This interface allows specific concrete animals to eat fresh food if they implement the IEatFresh interface.
```

Comment (selmaT273) [reply README.md]: Thank you for the feedback! Changed it locally.

File: README.md (hunk 1 above)
Comment (selmaT273) [reply README.md]: Thank you! Changed it locally.

**Pull request merged**